tutorials/tts/Evaluation_MelCepstralDistortion.ipynb

```diff
@@ -0,0 +1,646 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8b9b6e1c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## Import libraries\n",
+    "import IPython.display as ipd\n",
+    "import librosa\n",
+    "import numpy as np\n",
+    "import math\n",
+    "import matplotlib.pyplot as plt\n",
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "398b01e0",
+   "metadata": {},
+   "source": [
+    "## Defining Parameters for Spectrogram Generation\n",
+    "\n",
+    "In this section we define parameters required to generate our mel spectrograms. More information about these parameters can found in the Librosa documentation for [stft](https://librosa.org/doc/main/generated/librosa.stft.html), [mels](https://librosa.org/doc/main/generated/librosa.filters.mel.html) and [mfcc](https://librosa.org/doc/main/generated/librosa.feature.mfcc.html) generation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "191012de",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## Mel spec params\n",
+    "n_fft=1024\n",
+    "hop_length=256\n",
+    "win_length=None\n",
+    "window='hann'\n",
+    "n_mels = 80\n",
+    "sr = 22050\n",
+    "\n",
+    "## Mfcc params\n",
+    "n_mfcc=34"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "54830c8c",
+   "metadata": {},
+   "source": [
+    "A little bit about these parameters:  \n",
+    "    - `n_fft`: Number of fft_components or length of windowed signal after padding in STFT.  \n",
+    "    - `hop_length`: Number of audio samples between adjacent STFT columns.  \n",
+    "    - `window`: Window to use in STFT.  \n",
+    "    - `win_length`: Length of the window to be used.  \n",
+    "    - `n_mels`: Number of number of mel bands to generate.  \n",
+    "    - `sr`: Sample rate of the samples.  \n",
+    "    - `n_mfcc`: Number of MFCCs to generate."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4cb9e37c",
+   "metadata": {},
+   "source": [
+    "## Loading and Visualizing Data\n",
+    "\n",
+    "Lets apply the algorithm on a synthesized mel and ground truth mel pair for understanding.\n",
+    "\n",
+    "First, we we need a function to generate mel spectrograms from audio files. Mel spectrogram are generated using [librosa mel extractor](https://librosa.org/doc/main/generated/librosa.filters.mel.html)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "042ceb7f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def wav2mel(filename):\n",
+    "    \"\"\"\n",
+    "    Function to load an audio file and generate/return mel specs:\n",
+    "    Args:\n",
+    "        filename: Full path of the audio file.\n",
+    "    Returns:\n",
+    "        mels: Corresponding mel spectrogram.\n",
+    "    \"\"\"\n",
+    "    wav_, _ = librosa.load(filename)  # load() returns an (audio data, sample rate) tuple\n",
```

Comment:
You need to pass sr to librosa otherwise it auto resamples to 22050

Reply:
Done